1_network/src/network_dam_res_reaches.R

```diff
@@ -0,0 +1,114 @@
+#' @param zip char path to input zip file containing shapefile
+#' @param extract_dir char directory to unzip to, and read shapefile from
+#' @param out_ind char indicator file to represent output
+zipped_shp_to_rds <- function(zip, out_ind) {
+  extract_dir <- file.path(tempdir(), 
+                           tools::file_path_sans_ext(basename(zip)))
+  unzip(zip, exdir = extract_dir)
+  read_sf(extract_dir) %>% saveRDS(file = as_data_file(out_ind))
+  unlink(extract_dir, recursive = TRUE)
+  gd_put(out_ind)
+}
+
+#' @param dams_file directory of dams shapefile
+#' @param reservoirs_file directory of reservoir shapefile
+#' @param boundary polygon of basin boundaries to subset dams/reservoirs by
+#' @param outind output indicator file
+filter_dams_reservoirs_by_boundary <- function(dams_shp_ind, reservoirs_shp_ind, 
+                                               boundary_ind, out_ind) {
+  #dams are points, reservoirs are polygons
+  dams_shp <- readRDS(sc_retrieve(dams_shp_ind)) %>% 
+    st_transform(crs = 102039) %>% 
```

Comment:
Ok, Googling this a bit more and looking at the second warning, I think the EPSG code 102039 might be specific to ESRI or something?  I see the projection referenced as `EPSG:5070` in some other places.  Maybe whatever library defines EPSG codes is different on your machine, and does not include 102039?  It is what `st_crs` returns for the boundary shapefile though:
![image](https://user-images.githubusercontent.com/13987349/81591105-c4776280-9370-11ea-8df7-2fdf8777449a.png)

What does that return for you? 

In any case, I think using the CRS from the boundary shapefile rather than hardcoding it is probably the right choice.  You might end up with fighting indicator files though if the boundary shapefile is read differently though 😕

Reply:
Mine returns the same `102039`. What do you mean if the boundary shapefile is read differently? You mean if it returns a different ESPG?

Reply:
Well, your second warning seems to say that GDAL (which I assume `sf` is using under the hood here) doesn't have the 102039 projection defined.  This was not an issue for me, so I must have a different GDAL installed.  I guess the error happened in st_transform though instead of when the file is read.  I had thought maybe your machine would substitute a different projection when reading it.

Thinking about this a bit more, I bet that using st_transform(EPSG code) depends on GDAL having that EPSG code defined (yours apparently doesn't).  But if you use `st_transform(crs = sf::st_crs(boundary))` you are just using the projection definition stored in the boundary `object`, so GDAL having the projection doesn't matter.   Maybe there is a subtle difference in my GDAL's definition of the 102039 projection compared to what the `boundary` object uses, that would cause the object hashes to change with your PR?

TL;DR here is I think this is good to go with your change 😄

Reply:
Yes, you are correct that it happened in the transform -- I could read the file in just fine. 

Thanks for this follow-up, and for the note about potential changes in hashes and indicator files. That is giving me a clue to mystery rebuilds that we have in the pipeline.